### PR TITLE
add css and js file

### PR DESCRIPTION
**Pull request by faynald**
- **css** in folder css/styles.css

- **js** in folder js/javascript.js

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -5,11 +5,13 @@
...
+    <link rel="stylesheet" href="css/styles.css">
     <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-wEmeIV1mKuiNpC+IOBjI7aAzPcEZeedi5yW5f2yOq55WWLwNGmvvx4Um1vskeMj0" crossorigin="anonymous">
 </head>
 <body>
     
 </body>
 <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.2/dist/umd/popper.min.js" integrity="sha384-IQsoLXl5PILFhosVNubq5LC7Qb9DXgDA9i+tQ8Zj3iwWAwPtgFTxbJ8NT4GN1R8p" crossorigin="anonymous"></script>
 <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0/dist/js/bootstrap.min.js" integrity="sha384-lpyLfhYuitXl2zRZ5Bn2fqnhNAKOAaM/0Kr9laMspuaMiZfGmfwRNFh8HlMy49eQ" crossorigin="anonymous"></script>
+<script src="js/javascript.js"></script>
 </html>
```
Comment (NaMLiM) [inline index.php]: Ga ada newlinenya nih perbaikin dulu

**Review by NaMLiM — CHANGES_REQUESTED** [review]
Ganti Dulu

File: index.php (hunk 1 above)
Comment (faynald) [reply index.php]: sudah bos

**Review by NaMLiM — APPROVED** [review]
Ok All Done

**Comment by faynald** [comment]
> Ok All Done

bagus

**Pull request merged**